DragaliaAPI/Controllers/Dragalia/DragonController.cs

```diff
@@ -0,0 +1,510 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Numerics;
+using System.Security;
+using AutoMapper;
+using DragaliaAPI.Database.Entities;
+using DragaliaAPI.Database.Repositories;
+using DragaliaAPI.Database.Utils;
+using DragaliaAPI.Models.Generated;
+using DragaliaAPI.Services;
+using DragaliaAPI.Shared.Definitions;
+using DragaliaAPI.Shared.Definitions.Enums;
+using DragaliaAPI.Shared.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using static DragaliaAPI.Database.Utils.DragonConstants;
+
+namespace DragaliaAPI.Controllers.Dragalia;
+
+[Route("dragon")]
+[Consumes("application/octet-stream")]
+[Produces("application/octet-stream")]
+[ApiController]
```

Comment:
these are not needed when inheriting from DragaliaControllerBase (at least on the latest branches)